doc/getting_started.md

```diff
@@ -0,0 +1,36 @@
+# Getting started
+Now that you have successfully installed Azure CLI 2.0 and added the Azure DevOps extension, you are all set to get started!
+
+## Log in to Azure DevOps
+Before you can work with Azure DevOps, you need to log in to Azure CLI using the `az login` command.
+
+If the CLI can open your default browser, it will do so and load a sign-in page.
+Otherwise, you need to open a browser page and follow the instructions on the command line to enter an authorization code after navigating to https://aka.ms/devicelogin in your browser. For more information, please refer [azure cli login page](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli?view=azure-cli-latest).
+
+## Configuring defaults
+Although you can provide the organization and project for each command, we recommend you set these as defaults in configuration for seamless commanding. 

```

Comment:
do you also want to add that configuration made via `az configuration` are honored by CLI extension like output type?

Reply:
I would stay away from this for now. Will wait for some more feedback on this.

Reply:
What is the persona(s) which we think are going to consume these docs, if I think from a DevOps user (who is not a Azure User) then I think he will have a hard time if using this doc for getting started.
So knowing who is the consumer of these docs will be helpful

Reply:
Persona - a user who wants to use azure devops via CLI but has no clue about Azure CLI. So the expectation was to provide documentation to help the user get started using Azure DevOps extension. I was thinking configuring the output format may not be a P0 getting started requirement. Further, we are covering this in our samples section as well.